nixos/modules/programs/weylus.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.weylus;
+in
+{
+  options.programs.weylus = with types; {
+    enable = mkEnableOption "weylus";
+
+    openPorts = mkOption {
+      type = bool;
+      default = false;
+      description = ''
+        Open ports needed for the functionality of the program
+      '';
+    };
+
+     users = mkOption {
+      type = listOf str;
+      default = [ ];
+      description = ''
+        Weylus uses the uinput interface to simulate input events on Linux. To enable stylus and multi-touch support the user you're going to use must be added to this list. This allows them to synthesize input events system-wide, even when another user is logged in. Therefore, untrusted users should not be added to this list.
+      '';
+    };
+
+    package = mkOption {
```

Comment:
Are there any other options here?

Reply:
I mean if there is only one package what is the point of this option?

Reply:
Mostly flexibility. Maybe someone packages their own version but doesn't want to override the normal weylus package. 

I honestly just added it because I thought it was the norm, and I was expected to do that :shrug:.

I'd be open to removing it if you want

Reply:
I usually ask this just to make sure people understand overlays. 😄